Incorporate text editorial remarks. (#4589)
awsdocs/aws-doc-sdk-examples: resources/cdk/photo_asset_manager/README.md

# AWS CDK for PAM

This project will create the following in your AWS Cloud environment:

- AWS Identity and Access Management (IAM) group
- IAM user (added to the IAM group)
- Amazon Simple Storage Service (Amazon S3) buckets
- Amazon DynamoDB tables
- AWS Lambda function that performs image classification with Amazon Rekognition when new images are uploaded to the S3 bucket

> - Amazon API Gateway routing for Lambda functions

Reviewer: Using the "first appearance" names really reduces the readability, but I guess we can take it up with Marketing. 

As a reader, you're testing my patience by making me read out AWS Simple Storage Service and Amazon Identity Access Management.

Could we compromise with a hyperlink to the product page for each and keep the readable names? (IAM, S3)

Author: I sort of agree? Or at least I did at first. The more I thought about it though the more I realized this could be the first time the customer has every come across a term like API Gateway, and they might be unaware it's an Amazon product.

Reviewer 2: That's certainly the policy the docs org has taken for editing these documents. This isn't the first time anyone has seen these types of changes come through :)

There are three stacks that need to be deployed:

1. PamFrontEndInfraStack ({PAM_NAME}-FE-Infra-PAM) - Amazon Cloudfront distribution and S3 bucket host for static website files.
2. PamStack ({PAM_NAME}-{PAM_LANG}-PAM) - The backend resources (Amazon Rekognition, Lambda, Amazon S3, Amazon Cognito, API Gateway).
3. PamFrontEndAssetStack ({PAM_NAME}-FE-Assets-PAM) - The static website assets.

At this point you can now synthesize the AWS CloudFormation template for this code. Run `cdk ls` to see
a list of available stacks to synth/deploy.

```
cdk synth {PAM_NAME}-FE-Infra-PAM
```

> If everything looks good, go ahead and deploy. This step will make

Reviewer: ```suggestion
If everything looks good, proceed with `cdk deploy`. This step will make
```

Author: All of these changes direct from our editor.
